stock_landed_costs_average/model/stock_landed_costs.py

```diff
@@ -587,7 +588,13 @@ def compute_landed_cost(self, cr, uid, ids, context=None):
                         else:
                             value = (line.price_unit / total_line)
 
-                        if digits:
+                        # When the valuation is the last record, assign
+                        # line.price_unit less the accumulated value in
+                        # value_split
+                        if digits and \
+                                valuation == cost_adjust_lines[-1]:
```

Comment:
@JulioSerna because we have to know if ```valuation``` is the last register in ```cost_adjust_lines``` to assigned the remainder of ```line.price_unit```.